hmac-mapper/src/main/java/de/intension/validation/CustomClientValidationProviderFactory.java

```diff
@@ -1,25 +1,31 @@
 package de.intension.validation;
 
-import de.intension.protocol.oidc.mappers.HmacExtPairwiseSubMapper;
 import org.keycloak.models.KeycloakSession;
 import org.keycloak.validation.ClientValidationProvider;
 import org.keycloak.validation.ClientValidationProviderFactory;
 
+import de.intension.protocol.oidc.mappers.HmacPairwiseSubMapper;
+
 /**
- * Custom client validation factory to Support {@link de.intension.protocol.oidc.mappers.HmacExtPairwiseSubMapper} feature.
+ * Custom client validation factory to Support
+ * {@link de.intension.protocol.oidc.mappers.HmacPairwiseSubMapper} feature.
  */
-public class CustomClientValidationProviderFactory implements ClientValidationProviderFactory {
+public class CustomClientValidationProviderFactory
+    implements ClientValidationProviderFactory
+{
 
-    private final HmacExtPairwiseSubMapper hmacExtPairwiseSubMapper = new HmacExtPairwiseSubMapper();
-    private final ClientValidationProvider provider = new CustomClientValidationProvider(hmacExtPairwiseSubMapper);
+    private final HmacPairwiseSubMapper    hmacPairwiseSubMapper = new HmacPairwiseSubMapper();
+    private final ClientValidationProvider provider              = new CustomClientValidationProvider(hmacPairwiseSubMapper);
```

Comment:
this will always validate the newly created mapper object (default config). Not the one from the client. imo the validator classes can be deleted.